.env.template

```diff
@@ -0,0 +1,10 @@
+## ArxivDigest- environment seting 
+
+OPENAI_API_KEY=your_api_key
+
+
+
+## ArxivDigest- Envronment seting EMAIL SETTINGS
```

Comment:
Copy the warning in the readme into this file:

> **NOTE:** This is an important note! Do not edit and commit your .env.template with your personal key, it might reveal you API keys.